dev/ci/go-backcompat/flakefiles/v4.1.0.json

```
@@ -13,5 +13,10 @@
     "path": "github.com/sourcegraph/sourcegraph/internal/repos",
     "prefix": "EnqueueWebhookBuildJob",
     "reason": "Flakes repeatedly in CI"
+  },
+  {
+    "path": "github.com/sourcegraph/sourcegraph/internal/database",
+    "prefix": "WebhookList",
```

Pretty sure you need this:

```suggestion
    "prefix": "TestWebhookList",
```

(Which also means that the other flake above doesn't work?)

---

🤦🏻 thx!

Yep, TestWebhookCount shouldn't work as well, I'll check it during next CI run.